app/uk/gov/hmrc/agentclientrelationships/connectors/DesConnector.scala

```diff
@@ -50,24 +50,27 @@ object MtdItIdBusinessDetails {
 trait Relationship {
   val arn: Arn
   val endDate: Option[LocalDate]
+  val dateFrom: Option[LocalDate]
 }
 
-case class ItsaRelationship(arn: Arn, endDate: Option[LocalDate]) extends Relationship
+case class ItsaRelationship(arn: Arn, endDate: Option[LocalDate], dateFrom: Option[LocalDate]) extends Relationship
```

Comment:
we need either endDate, startDate or dateFrom, dateTo

Reply:
:+1: